atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -10,9 +10,17 @@
 try:
     # py2
     from ConfigParser import ConfigParser
+    # Although io.StringIO is available in python2, it
+    # always expects unicode during write. Since
+    # ConfigParser.ConfigParser does not support
+    # writing unicode, io cannot be used.
+    from StringIO import StringIO
```

Comment:
Importing ```BytesIO``` as ```StringIO``` might also work (I did that hack at https://github.com/projectatomic/atomic-reactor/blob/master/atomic_reactor/plugins/pre_add_yum_repo_by_url.py#L37)